test/test_fit_transform.py

```diff
@@ -63,6 +64,8 @@ def test_lmnn(self):
     assert_array_almost_equal(res_1, res_2)
 
   if has_installed_skggm():
+    # TODO: remove the mark when SDML has become deterministic
```

Comment:
SDML is not deterministic (see skggm issue https://github.com/skggm/skggm/issues/121), so for now I exclude it from this test, until we find a solution (I don't think it should prevent merging this PR though)